packages/core/src/render3/util.ts

```diff
@@ -100,6 +100,8 @@ export function getNativeByTNode(tNode: TNode, hostView: LView): RElement|RText|
 }
 
 export function getTNode(index: number, view: LView): TNode {
+  assertGreaterThan(index, -1, 'wrong index for TNode');
+  assertLessThan(index, view[TVIEW].data.length, 'wrong index for TNode');
```

Comment:
```suggestion
  ngDevMode && assertLessThan(index, view[TVIEW].data.length, 'wrong index for TNode');
```

Thanks to the symbol updates which showed these assertion functions popping up.

Reply:
good catch